availability-store/src/lib.rs

```diff
@@ -1,18 +1,18 @@
-// Copyright 2018 Parity Technologies (UK) Ltd.
-// This file is part of Polkadot.
+// Copyright 2017-2019 Parity Technologies (UK) Ltd.
```

Comment:
Looks like this faile was created in 2018. Shouldn't be:
```suggestion
// Copyright 2018-2019 Parity Technologies (UK) Ltd.
```

?

Reply:
It's a script that streamlines the copyright headers. The benefit is you can ensure all files are prepended with a correct header. Downside is the actual year when the file was created is no longer part of the header. Would you insist on leaving the year untouched?

Reply:
We have a script that updates every year from `x` to `x - current_year()` or `x - y` to `x - current_year`

Reply:
No, I don't insist. But I recommend checking with the rest of the team.

at least in substrate, I am pretty sure that we maintain the creation date of a file persistently (or at least we try until the codebase is too large to maintain like that).

And I assume the script can actually maintain the creation date right? Just have to scrape the year and check if it needs a bump or not. If it has the format `X-Y` update `Y` else, append a `-Y`.